django/utils/cache.py

```diff
@@ -443,3 +444,35 @@ def _to_tuple(s):
     if len(t) == 2:
         return t[0].lower(), t[1]
     return t[0].lower(), True
+
+
+def invalidate_view_cache(path=None, request=None, key_prefix=None, cache=None):
```

Comment:
I wonder if we want just a `request` parameter, which could be a path, then converted in the same code block as below? 🤔